lib/src/components/NxList/__tests__/NxListItem.test.tsx

```diff
@@ -5,35 +5,48 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
+
 import NxList from '../NxList';
-import { getMountedComponent, getShallowComponent } from '../../../__testutils__/enzymeUtils';
-import 'jest-enzyme';
-
-describe('NxListItem', function() {
-
-  const minimalProps = {};
-  const getShallow = getShallowComponent(NxList.Item, minimalProps);
-  const getMounted = getMountedComponent(NxList.Item, minimalProps);
-
-  it('renders the children in an .nx-list__item', function() {
-    const children = [
-      <NxList.Text key="1">Test Item 1 Text</NxList.Text>,
-      <NxList.Subtext key="2">Test Item 1 Subtext</NxList.Subtext>
-    ];
-    const contentEl = getMounted({children});
-    expect(contentEl).toExist();
-    expect(contentEl).toContainMatchingElements(2, 'span');
-    expect(contentEl.find('li')).toHaveClassName('nx-list__item');
-    expect(contentEl.find('span').at(0)).toHaveText('Test Item 1 Text');
-    expect(contentEl.find('span').at(0)).toHaveClassName('nx-list__text');
-    expect(contentEl.find('span').at(1)).toHaveText('Test Item 1 Subtext');
-    expect(contentEl.find('span').at(1)).toHaveClassName('nx-list__subtext');
 
+describe('NxList.Item', function() {
+  const quickRender = rtlRender(NxList.Item, {}),
+      renderEl = rtlRenderElement(NxList.Item, {});
+
```

Comment:
Should test that the top-level element is a `listitem`.

Reply:
Added here 999143c7e42cc36bc438640defd458e8ed027d0a